### PR TITLE
Remove ShARC from transferring and choosing resources

### PR DESCRIPTION
**Pull request by Carldkennedy**
Removes ShARC from 'Transferring files', 'Choosing appropriate resources' and 'mfa tips' sections

### CODE REVIEW

File: hpc/Choosing-appropriate-resources.rst [hunk 1]
```diff
@@ -117,7 +103,7 @@ The time used by a job is typical quantified into 2 values by the scheduler:
 * the **"wallclock"** (the time your job took if measured by a clock on your wall)
 * the consumed **"CPU time"** (a number of seconds of compute, derived directly from the amount of CPU time used on all cores of a job).
 
-How to determine these values can be seen below using the :ref:`seff` or :ref:`qacct` commands as below:
+How to determine these values can be seen below using the :ref:`seff` command as below:
 
 .. tabs::
```
Comment (willfurnass) [inline hpc/Choosing-appropriate-resources.rst:109]: Do we need separate tabs for Stanage vs Bessemer?

File: hpc/Choosing-appropriate-resources.rst [hunk 2]
```diff
@@ -200,11 +173,11 @@ In order to determine your CPU requirements, you should investigate if your prog
 
 If your job / program supports multiple cores, you need to assess whether it supports SMP (symmetric multiprocessing) where you can only use CPUs on 1 node or MPI (message passing interface) where you can access as many nodes, CPUs and cores as are available.
 
-For SMP only type parallel processing jobs: you can use a maximum of 64 cores on Stanage, 40 cores on Bessemer and 16 cores on ShARC. Ideally you should use Stanage or Bessemer as you can not only access more cores, you are using more modern cores.
+For SMP only type parallel processing jobs: you can use a maximum of 64 cores on Stanage and 40 cores on Bessemer.
```
Comment (willfurnass) [inline hpc/Choosing-appropriate-resources.rst:176]:
```suggestion
For single-node jobs: you can use a maximum of 64 cores on Stanage and 40 cores on Bessemer. 
```

File: hpc/Choosing-appropriate-resources.rst [hunk 3]
```diff
@@ -200,11 +173,11 @@ In order to determine your CPU requirements, you should investigate if your prog
...
 
-For SMP only type parallel processing jobs: you can use a maximum of 64 cores on Stanage, 40 cores on Bessemer and 16 cores on ShARC. Ideally you should use Stanage or Bessemer as you can not only access more cores, you are using more modern cores.
+For SMP only type parallel processing jobs: you can use a maximum of 64 cores on Stanage and 40 cores on Bessemer. 
 
-For multiple node MPI type parallel processing jobs: these can run on both Stanage and ShARC and although you can access as many cores as are available you must consider how long a job will take to queue waiting for resources compared the the decrease in time for the job to complete computation.
+For multiple node MPI type parallel processing jobs: these can run on Stanage and although you can access as many cores as are available you must consider how long a job will take to queue waiting for resources compared the the decrease in time for the job to complete computation.
```
Comment (willfurnass) [inline hpc/Choosing-appropriate-resources.rst:178]:
```suggestion
For multiple-node MPI-type parallel processing jobs: these can run on Stanage and although you can access as many cores as are available you must consider how long a job will take to queue waiting for resources compared the the decrease in time for the job to complete computation.
```

File: hpc/Choosing-appropriate-resources.rst [hunk 4]
```diff
@@ -200,11 +173,11 @@ In order to determine your CPU requirements, you should investigate if your prog
...
 
-Single node MPI type parallel jobs can run on ShARC (when running in the SMP parallel environment), Stanage and Bessemer.
+Single node MPI type parallel jobs can run on Stanage and Bessemer.
```
Comment (willfurnass) [inline hpc/Choosing-appropriate-resources.rst:180]:
```suggestion
Single-node MPI-type parallel jobs can run on Stanage and Bessemer.
```

File: hpc/Choosing-appropriate-resources.rst [hunk 5]
```diff
@@ -334,10 +271,10 @@ Determining memory requirements:
...
 
 
-**By using the emailing parameters of the qsub or sbatch command:**
+**By using the emailing parameters of the sbatch command:**
 
 
-Submit your job ``qsub`` or ``sbatch`` by specifying very generous memory and time requirements to ensure that it runs to completion" and also using the ``-M`` and ``-m abe`` or  ``--mail-user=`` and ``--mail-type=ALL``   parameters to receive an email-report. The mail message will list the maximum memory usage ( maxvmem / MaxVMSize  ) as well as the wallclock time used by the job.
+Submit your job with ``sbatch`` by specifying very generous memory and time requirements to ensure that it runs to completion" and also using the ``-M`` and ``-m abe`` or  ``--mail-user=`` and ``--mail-type=ALL``   parameters to receive an email-report. The mail message will list the maximum memory usage ( maxvmem / MaxVMSize  ) as well as the wallclock time used by the job.
 
 .. tabs::
```
Comment (willfurnass) [inline hpc/Choosing-appropriate-resources.rst:279]: Do we need tabs here now ShARC has gone?

File: hpc/transferring-files.rst [hunk 6]
```diff
@@ -195,20 +195,28 @@ way of getting files onto the clusters. Using your home connection will be a sig
 speed bottleneck compared to large amounts of download bandwidth available on the clusters.
 Directly downloading to the cluster avoids this bottleneck!
 
-Using a qsh-vis session
-^^^^^^^^^^^^^^^^^^^^^^^
+Using Firefox Browser
```
Comment (willfurnass) [inline hpc/transferring-files.rst:198]: Put _after_ the section on wget/curl?

**Review by willfurnass — APPROVED** [review]
Suggested some small changes

**Review by willfurnass — APPROVED** [review]

**Review by jkwmoore — APPROVED** [review]
Feels a bit Jerry Smith approving my own changes:
image

But LGTM everywhere. Ty! Carl.

**Pull request merged**